benchmark/opperf/utils/profiler_utils.py

```diff
@@ -228,10 +232,16 @@ def python_profile(func):
 
     @functools.wraps(func)
     def python_profile_it(*args, **kwargs):
-        start_time = time.perf_counter()    # 1
-        res = func(*args, **kwargs)
-        end_time = time.perf_counter()      # 2
-        run_time = end_time - start_time    # 3
+        runs = args[1]
+        modified_args = (args[0], 1, args[2])
```

Comment:
Can you give an example case what are now sent in modified_args ?

Reply:
Currently when you run opperf
args are - `[op, runs, args]`
runs  is the # of times we want to run this operator

For python profiler to work, we need to call the operator explicitly those many times
Hence runs = args[1] stores the number of times user wanted to run the operator

Now, the alias takes the argument to run MXNet profiler on it
We're overwriting on that, hence we pass run=1 to MXNet profiler
modified args = `[op, 1, args]`
Does that answer your question?